__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require("supertest")
+const app = require("../app")
+const db = require("../db/connection")
+const data = require("../db/data/test-data")
+const seed = require("../db/seeds/seed")
+
+
+
+beforeEach(() => {
+    return seed(data);
+})
+
+afterAll(() =>  {
+    return db.end();
+})
+
+describe('/api/topics', ()    =>  {
+    describe('GET', ()   =>   {
+        it('Status code 200', () =>  {
+            return request(app).get("/api/topics").expect(200);
+        });
+        it('should respond with an array of topic objects', ()   =>  {
+            return request(app)
+            .get("/api/topics")
+            .then(({body}) => {
+            const topics = body.topics;
+            expect(Array.isArray(topics)).toBe(true)
+            topics.forEach((topic) => {
+                expect(typeof topic.slug).toBe('string');
+                expect(typeof topic.description).toBe('string')
+            });
+          });
+        });
+    }); 
+});
+
```

Comment:
Please think of an error you may be able predict early on. what happens if you go to the wrong path?